src/cmd-cloud-prune

```diff
@@ -162,7 +174,24 @@ def main():
                         case "build":
                             prune_build(s3_client, bucket, prefix, build_id, args.dry_run)
                             pruned_build_ids.append(build_id)
-
+                        case "containers":
+                            # Only prune for x86_64 as it's required for all builds, covering all architectures.
+                            if arch == "x86_64":
+                                if build_id in barrier_releases:
+                                    print(f"Release {build_id} is a barrier release, keeping.")
+                                    continue
+                                # Retrieve container tags excluding the stream name since it updates with each release.
+                                container_tags = get_container_tags(meta_json, exclude=[stream])
+                                if container_tags:
+                                    containers_config = {
+                                        "dry_run": args.dry_run,
+                                        "repository_url": container_tags.get("repository_url", ""),
+                                        "registry_auth_file": args.registry_auth_file,
+                                    }
+                                    for tag in container_tags["tags"]:
+                                        prune_container(tag, containers_config)
+                                else:
+                                    print(f"No container tags to prune for build {build_id} on architecture {arch}.")
```

Comment:
Since we would only ever hit this in the `x86_64` case.

```suggestion
                                    print(f"No container tags to prune for build {build_id}.")
```

Reply:
fixed in new upload